go/test/endtoend/vault/dbcreds_secret.json

```diff
@@ -1,17 +1,23 @@
 {
+  "root": [
+    "RootPass"
+  ],
   "vt_app": [
-    "password"
+    "VtAppPass"
```

Comment:
changing these passwords so I can reuse GetPasswordUpdateSQL. This will make test consistent as well ...